examples/fiverthirtyeight-example/pages/index.tsx

```diff
@@ -82,6 +98,16 @@ export function DesktopItem({dataset} : { dataset: Dataset}) {
               </a>
             )}
           </td>
+          <td className="py-8">
```

Comment:
I think we should separate the button a bit more. Maybe add a ml-1 or ml-2 to this button